package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -516,23 +431,26 @@ def __init__(self, filename, n_atoms=None, mmap=None, **kwargs):
                           "is used!".format(n_atoms, self.n_atoms))
                 raise ValueError(errmsg)
         except KeyError:
-            errmsg = ("NCDF trajectory {0} does not contain atom "
-                      "information".format(self.filename))
+            errmsg = ("NCDF {0} {1} does not contain atom "
+                      "information".format(file_format, self.filename))
             raise ValueError(errmsg) from None
 
-        try:
-            self.n_frames = self.trjfile.dimensions['frame']
-            # example trajectory when read with scipy.io.netcdf has
-            # dimensions['frame'] == None (indicating a record dimension that
-            # can grow) whereas if read with netCDF4 I get
-            # len(dimensions['frame']) ==  10: in any case, we need to get
-            # the number of frames from somewhere such as the time variable:
-            if self.n_frames is None:
-                self.n_frames = self.trjfile.variables['coordinates'].shape[0]
-        except KeyError:
-            errmsg = (f"NCDF trajectory {self.filename} does not contain "
-                      f"frame information")
-            raise ValueError(errmsg) from None
+        # try:
```

Comment:
Please clean up the commented-out code or add a comment explaining why it’s being retained.